package.json

```diff
@@ -93,7 +93,7 @@
     "scroll-into-view": "^1.3.1",
     "seamless-immutable": "^6.0.1",
     "showdown": "^1.6.4",
-    "sinon": "^1.17.3",
+    "sinon": "^2.3.5",
```

Comment:
The `yarn.lock` file should have changed when you ran this command but isn't changed in this commit. Can you add the updated lockfile as well?